### DISCUSSION TITLE
How to use digest auth with RTSP ?

### DISCUSSION THREAD

**Discussion by KaleiRo** [bluenviron/mediamtx#786] [Questions]
Hi,

I successfully connect RTSP camera with basic auth (credential included in URL) but when you can't use basic auth and only digest, how you do?

Thanks.

**Comment by aler9** [comment]
Hello, during the handshake between a RTSP client and a camera, the camera tells the client which authentication methods are supported. The client then uses one of the available methods. If only digest is available, the client uses digest (at least, rtsp-simple-server does it).

[a comment by an automated account is not shown]
